### PR TITLE
Design Implementation

### PR DESCRIPTION
**Pull request by ogzhanolguncu**

### CODE REVIEW

File: packages/react-databrowser/src/components/databrowser/components/data-display-container/data-delete.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+export const DataDelete = ({ onDataKeyChange, selectedDataKey }: Props) => {
+  const { toast } = useToast();
+  const deleteKey = useDeleteKey();
+
+  const handleDeleteKey = async () => {
+    try {
+      const result = await deleteKey.mutateAsync(selectedDataKey);
+      if (result) {
+        onDataKeyChange(undefined);
```
Comment (fahreddinozcan) [inline packages/react-databrowser/src/components/databrowser/components/data-display-container/data-delete.tsx:21]: did we forget this as undefined?

File: packages/react-databrowser/src/components/databrowser/components/data-display-container/display-scrollarea.tsx [hunk 2]
```diff
@@ -1,19 +1,20 @@
 import { ScrollArea } from "@/components/ui/scroll-area";
 import { CopyToClipboardButton, handleCopyClick } from "@/components/databrowser/copy-to-clipboard-button";
+import parse from "html-react-parser";
+import formatHighlight from "@/lib/utils";
 
 export const DisplayScrollarea = ({ data }: { data: string | JSON | null }) => {
   const stringifiable = toJsonStringifiable(data);
+  console.log({ data });
```
Comment (fahreddinozcan) [inline packages/react-databrowser/src/components/databrowser/components/data-display-container/display-scrollarea.tsx]: Let's remove this console.log

File: packages/react-databrowser/src/components/databrowser/components/data-display-container/data-delete.tsx (hunk 1 above)
Comment (ogzhanolguncu) [inline packages/react-databrowser/src/components/databrowser/components/data-display-container/data-delete.tsx:21]: You need to reset a chosen key to refresh it from the sidebar. When you delete a key, the selected item disappears from the left side, which then resets the right side. You can even try it out yourself.

**Comment by fahreddinozcan** [comment]
Okey makes sense, I grasped it a bit further into the code

**Comment by fahreddinozcan** [comment]
> You need to reset a chosen key to refresh it from the sidebar. When you delete a key, the selected item disappears from the left side, which then resets the right side. You can even try it out yourself. 

Thanks, I grasped it a bit further into the code. Np

**Review by fahreddinozcan — APPROVED** [review]

**Pull request merged**